Build-and-Deploy-Your-First-Dashboard.mdx

```diff
@@ -0,0 +1,420 @@
+
+To move a chart simply drag and drop the chart.
+
+#### Add dashboard filters
+
+To add a dashboard filter please click on **New Filter**, 
+
+<AccordionGroup>
+<Accordion title="1. Add Filter">
+Click on the **+ aggregations** , this will create ```SQL SELECT * FROM "aggregations" ```   
+in the SQL editor.
+
+Click on **Preview data** to run the SQL and **Next** once you see the preview data.
+     <Frame>
+
+   <Frame>
+
+  <img
+  className="block dark:hidden"
+  src="/images/build-your-first-dashboard/light/example-dashboard-filter-dark.png"
+  alt="Hero Light"
+/>
+<img
+  className="hidden dark:block"
+  src="/images/build-your-first-dashboard/dark/example-dashboard-filter-dark.png"
+  alt="Hero Dark"
+/>
+     </Frame>
+    </Frame>
+
+
+
+  </Accordion>
+
+  <Accordion title="2. Set filter type">
+
+
+
+   <Frame>
+
+  <img
+  className="block dark:hidden"
+  src="/images/build-your-first-dashboard/light/example-dashboard-filter-dark.png"
+  alt="Hero Light"
+/>
+<img
+  className="hidden dark:block"
+  src="/images/build-your-first-dashboard/dark/example-dashboard-filter-dark.png"
+  alt="Hero Dark"
+/>
+     </Frame>
+
+
+
+  </Accordion>
+
+    <Accordion title="2. Select charts">
+
+
+
+   <Frame>
+
+  <img
+  className="block dark:hidden"
+  src="/images/build-your-first-dashboard/light/example-dashboard-filter-dark.png"
+  alt="Hero Light"
+/>
+<img
+  className="hidden dark:block"
+  src="/images/build-your-first-dashboard/dark/example-dashboard-filter-dark.png"
+  alt="Hero Dark"
+/>
+     </Frame>
+
+
+
+  </Accordion>
+    <Accordion title="4. Filter Preview">
+
+
+
+   <Frame>
+
+  <img
+  className="block dark:hidden"
+  src="/images/build-your-first-dashboard/light/example-dashboard-filter-dark.png"
+  alt="Hero Light"
+/>
+<img
+  className="hidden dark:block"
+  src="/images/build-your-first-dashboard/dark/example-dashboard-filter-dark.png"
+  alt="Hero Dark"
+/>
+     </Frame>
+
+
+
+  </Accordion>
+</AccordionGroup>
+
+<Tip>For more information on dashboard filters, check out our [Dashboard Filters Guide](/filters/filters-in-depth).</Tip>
+
+
+
+#### Add charts
+
+To add additional charts you can go to through the steps of creating a new chart. 
+
+Alternatively if you have already built a chart you can click on **Add Chart** to use on your dashboard.
+ <Frame>
+
+  <img
+  className="block dark:hidden"
+  src="/images/build-your-first-dashboard/light/add-chart-dark.png"
+  alt="Hero Light"
+/>
+<img
+  className="hidden dark:block"
+  src="/images/build-your-first-dashboard/dark/add-chart-dark.png"
+  alt="Hero Dark"
+/>
+     </Frame>
+
+
+
+
+## Share your dashboard
+
+You can share your dashboard via email with anyone in by clicking on the share button and sharing it as follows, if you would like a preview of the dashboard please click on the preview.
+
+Your team members will be invited to your dashboard. By sharing your dashboard they will become a team member and can collaborate on future dashboards with you.
```

Comment:
Rewrite this sentence to:

By sharing your dashboard, they will become a team member of your organization and can view, edit and collaborate on the dashboard(s) with you.